src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsEsaCaseData.java

```diff
@@ -61,4 +62,11 @@ public List<String> getSchedule3Selections() {
         }
         return null;
     }
+
+    @JsonIgnore
+    public DynamicList defaultEsaRegulationsYears() {
+        final DynamicListItem dynamicListItem2008 = new DynamicListItem("2013", "2013");
+        return new DynamicList(dynamicListItem2008, List.of(new DynamicListItem("2008", "2008"), dynamicListItem2008));
+    }
```

Comment:
I guess this is the correct CCD type that works rather than e.g. `DynamicRadioList`?